sharepoint/sharepoint-ps/sharepoint-online/Set-SPOCopilotPromoOptInStatus.md

```diff
@@ -0,0 +1,59 @@
+---
+external help file: sharepointonline.xml
+Module Name: Microsoft.Online.SharePoint.PowerShell
+online version: https://learn.microsoft.com/powershell/module/sharepoint-online/Set-SPOCopilotPromoOptInStatus
+applicable: SharePoint Online
+title: Set-SPOCopilotPromoOptInStatus.md
+schema: 
+author: swathi.iruvanti
+ms.author: siruvanti
+ms.reviewer:
+---
+# Set-SPOCopilotPromoOptInStatus
+
+## SYNOPSIS
+
+Retrieves the IsCopilotPromoStatusEnabled ( "true/false" ) state of the Copilot promo opt-in bit and stores the information.
```

Comment:
this cmdlet does "set" -- it doesn't "retrieve" -- so change this to something like

```
Sets the Opt-In Copilot promo status for the tenant.
```